accounts/forms.py

```diff
@@ -396,6 +410,11 @@ def __init__(self, request, *args, **kwargs):
         self.fields['sound_signature'].widget.attrs['class'] = 'unsecure-image-check'
         self.fields['is_adult'].widget.attrs['class'] = 'bw-checkbox'
         self.fields['not_shown_in_online_users_list'].widget = forms.HiddenInput()
+        self.fields['ui_theme_preference'].label = "User Interface theme preference:"
+
+        # If user has no sounds, do not show the AI sound usage preference field
+        if request.user.profile.num_sounds == 0:
+            self.fields.pop('ai_sound_usage_preference')
```

Comment:
does this mean that the process for a user to change their preference means that they first need to upload a sound and then go back to their profile? Can we make it clearer somehow?

Reply:
Aaaaand we could email uploaders about that. It will be ~38k emails, so should be doable...